lite-rpc/src/lib.rs

```diff
@@ -33,7 +33,7 @@ pub const DEFAULT_TRANSACTION_CONFIRMATION_STATUS: TransactionConfirmationStatus
     TransactionConfirmationStatus::Finalized;
 
 #[from_env]
-pub const DEFAULT_GRPC_ADDR: &str = "http://127.0.0.0:10000";
+pub const DEFAULT_GRPC_ADDR: &str = "http://localhost:10000";
```

Comment:
Should be 127.0.0.1 i think there was an issue before if we specify localhost for websockets or grpc, don't remember. But if you confirm it is fixed now then you can keep it.

Reply:
The issue was with 127.0.0.1. For example on amd81 server it doesn't work. That why I change to localhost.